deploy-cockroachdb-on-aws.md

```diff
@@ -69,22 +69,37 @@ You can create these rules using [Security Groups' Inbound Rules](http://docs.aw
 | Port Range | **26257** |
 | Source | Your application's IP ranges |
 
-To connect your application to CockroachDB, use a [PostgreSQL wire protocol driver](install-client-drivers.html).
-
-## Step 2. Create Instances
+## Step 2. Create instances
 
 [Create an instance](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/launching-instance.html) for each node you plan to have in your cluster. We [recommend](recommended-production-settings.html#cluster-topology):
 
 - Running at least 3 nodes to ensure survivability.
 - Selecting the same continent for all of your instances for best performance.
 
-## Step 3. Generate Your Certificates
+## Step 3. Set up load balancing
+
+Each CockroachDB node is an equally suitable SQL gateway to your cluster, but to ensure client performance and reliability, it's important to use TCP load balancing:
+
+- **Performance:** Load balancers spread client traffic across nodes. This prevents any one node from being overwhelmed by requests and improves overall cluster performance (queries per second).
+
+- **Reliability:** Load balancers decouple client health from the health of a single CockroachDB node. In cases where a node fails, the load balancer redirects client traffic to available nodes.
+
+AWS offers fully-managed load balancing to distribute traffic between instances.
+
+1. 	[Add AWS load balancing](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-increase-availability.html). Be sure to:
+	- Set forwarding rules to route TCP traffic from the load balancer's port **26257** to port **26257** on the node Droplets.
+	- Configure health checks to use HTTP port **8080** and path `/health`.
```

Comment:
this section is for secure mode. The AWS load balancer must be told to use https